task-launcher/src/tasks/shared/trials/afcStimulus.js

```diff
@@ -536,6 +537,7 @@ export const afcStimulus = ({ trialType, responseAllowed, promptAboveButtons, ta
         promptAboveButtons: promptAboveButtons,
         task: task,
         layoutConfig,
```

Comment:
layoutConfig is an object which has layout info, so you can pass `showPrompt` as a key to that object.